README.md

```
@@ -14,9 +14,16 @@ _Mokksy_ - Mock HTTP Server, built with [Kotlin](https://kotlinlang.org/) and [K
 * [Key Features](#key-features)
 * [Quick start](#quick-start)
 * [Responding with predefined responses](#responding-with-predefined-responses)
-  * [GET Request](#get-request)
-  * [POST Request](#post-request)
-* [Server-Side Events (SSE) Response](#server-side-events-sse-response)
+  * [GET request](#get-request)
+  * [POST request](#post-request)
+* [Server-Side Events (SSE) response](#server-side-events-sse-response)
```

markdownlint-cli2 (0.21.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 0

(MD005, list-indent)